frontend/src/components/object/DeleteObjectButton.vue

```diff
@@ -25,16 +30,19 @@ const props = defineProps({
 
 // Deletion
 const displayNoFileDialog = ref(false);
-const confirmDelete = () => {
+const confirmDelete = async () => {
```

Comment:
Does this need to be declared as an async function? Looks like the only await in this scope is the await in the accept function, which is already in its own async closure.

Reply:
Correct, not needed. Fixed